.github/scripts/gh-thread.sh

```
@@ -0,0 +1,67 @@
+case "$CMD" in
+  list)
+    PR="${1:?usage: gh-thread list <pr-number>}"
+    case "$PR" in
+      *[!0-9]*) echo "gh-thread: pr-number must be numeric, got: $PR" >&2; exit 64 ;;
+    esac
+    gh api graphql \
+      -f query='query($o:String!,$r:String!,$n:Int!){
+        repository(owner:$o,name:$r){pullRequest(number:$n){
+          reviewThreads(first:100){nodes{
+            id isResolved path line
+            comments(first:50){nodes{author{login} body url}}}}}}}' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)